stdlib/2and3/distutils/dist.pyi

```diff
@@ -1,7 +1,11 @@
 # Stubs for distutils.dist
+from distutils.cmd import Command
 
-from typing import Any, Mapping, Optional
+from typing import Any, Mapping, Optional, Dict, Tuple, Union, List
 
 
 class Distribution:
     def __init__(self, attrs: Optional[Mapping[str, Any]] = ...) -> None: ...
+    def get_option_dict(self, command: str) -> Dict[str, Tuple[str, str]]: ...
+    def parse_config_files(self, filenames: Optional[List[str]] = ...) -> None: ...
```

Comment:
Use `Iterable` instead of `List` here, since any iterable works. Also filenames can be `Text`.